### DISCUSSION TITLE
How to place legal details on my Modoboa installation

### DISCUSSION THREAD

**Discussion by sven-borkert** [modoboa/modoboa#3048] [Q&A]
Hi,

there is a big German provider (@t-online.de) that does not allow you to send mails to their customers, unless your mailserver has a webpage with legal details and contact information. 

I used the Moboboa installer to set up my installation, this worked fine. But I have a hard time to find any way to place a webpage on the root of the domain, because it keeps redirecting me to the login of the webinterface.

What would be the easiest way to place a static webpage on the root of the mailservers domain?

Regards,
Sven

**Comment by H-A-L-I-S** [comment]
Just a suggestion, if your information is true (which I highly doubt): Tell your contacts that (@t-online.de) does not accept your mails and ask them to use another service without these non-standard restrictions.

There are plenty of mail servers without a website at all.

Although, I doubt your information is correct. If you can not send mails to @t-online.de, you might have another issue

**Comment by sven-borkert** [comment]
Hi,

I cannot tell my user that he cannot send mails to an important customer, or tell the customer he should use a different provider. This is actually a known issue, when you search for "t-online email 554" you will find lots of german discussions about this (See bottom of this post for an example). T-Online seems to be the only provider so far behaving like that.

This is what you get when you try sending an email to a receipient @t-online.de:

```
[<xxxx@t-online.de>](mailto:xxxx@t-online.de): host mx01.t-online.de[194.25.134.72]
refused to talk to me: 554 IP=x.x.x.x - None/bad reputation. Ask your
postmaster for help or to contact [tobr@rx.t-online.de](mailto:tobr@rx.t-online.de) for reset. (NOWL)
```

My server was not listed on any IP blacklist at that time.

When you get in contact with them and ask about the reason for the reject, this is the answer (I translated it):

```
After **we allow only traceable commercial and comparable
operators to connect to our mail servers**,
please use an SMTP relay or mail gateway of the hoster or
ISP to send e-mails within the scope of the contractual services from the mail server
via their official mail gateway. The support there will
will be happy to help you with the configuration.

From our point of view, however, a host would be understandably
commercial, if the requirements of section 4.1 of our FAQ at
<https://postmaster.t-online.de/#t4.1> would be fulfilled.
```

They really mean it, no joke. I then put a page with my name, address and all contact data on my server, told them about it, and they confirmed it's ok now and they put my server on their whitelist, then the problem was solved.

Regards,
Sven

https://forum.netcup.de/administration-eines-server-vserver/vserver-server-kvm-server/16789-t-online-blockt-meine-mails/

**Reply by H-A-L-I-S** [reply]
There are user claiming this to be the case, but the T-Online writes in their policy, that they want legal information from senders of mass mails. I assume your issue lies somewhere else, not in the  missing legal information
